src/Estimators/ReferencePointsInput.h

```diff
@@ -0,0 +1,104 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2022 QMCPACK developers.
+//
+// File developed by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Laboratory
+//
+// Some code refactored from: ReferencePoints.h & ReferencePoints.cpp
+//////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef QMCPLUSPLUS_REFERNCE_POINTS_INPUT_H
+#define QMCPLUSPLUS_REFERNCE_POINTS_INPUT_H
+
+#include <map>
+#include <string>
+#include <string_view>
+
+#include <Configuration.h>
+#include <OhmmsPETE/Tensor.h>
+#include <OhmmsPETE/TinyVector.h>
+#include "InputSection.h"
+
+namespace qmcplusplus
+{
+namespace testing
+{
+template<typename T>
+class EnergyDensityTests;
```

Comment:
I don't see it's been consumed in this file. Is there a better place to host these lines?